pkg/arvo/app/graph-store.hoon

```diff
@@ -826,11 +831,79 @@
       :+  %add-nodes
         [ship term]
       %-  ~(gas by *(map index:store node:store))
-      %+  turn  (tap:orm `graph:store`(subset:orm p.children.u.node end start))
+      %+  turn  (tap:orm `graph:store`(lot:orm p.children.u.node end start))
       |=  [=atom =node:store]
       ^-  [index:store node:store]
       [(snoc index atom) node]
     ==
+  ::
+      [%x %deep-nodes-older-than @ @ @ @ ~]
```

Comment:
Flagging this ship parsing code as something that should be refactored next sprint

Reply:
What about the ship parsing code needs to be refactored?

Reply:
I'd consider moving the resource parsing out to a `|^`
```hoon
[%graph @ @ *]
=/  =ship  (slav %p i.t.path)
=/  =term  i.t.t.path
|^
?+  t.t.t.path  [~ ~]
   ~   (get-graph ship term)
   [%subset @ @]
  =/  start=@ud  i.t.path
  =/  end=@ud   i.t.t.path
  :: subset
   [%node *]
  ::  node
```

etc. etc.
Mainly for conciseness and readability. (You don't have overly long t.t.t.t.path accesses)

Reply:
Oh I see. I hadn't considered using that pattern, but I like it! Happy to refactor in that way